deps/readline.lua

```diff
@@ -396,7 +396,7 @@ local keyHandlers =
     self:getHistory(10)
   end},
   -- Printable characters
-  {{function(key, char) return char > 31 and key:sub(1,1) or nil end}, function(self, consumedKeys)
+  {{function(key, char) return char > 31 and key or nil end}, function(self, consumedKeys)
```

Comment:
Since `sub` was defined as `local sub = ustring.sub`, I assumed that it would account for the unicode characters. Does `ustring.sub` not work when called on a regular string?